### PR TITLE
UI style to display team members

### PR DESCRIPTION
**Pull request by fikrialwan**
- [x] Display main committee team member on team page
- [x] Display developer team member on team page

### CODE REVIEW

File: app/components/shared/team-members.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+	}
+
+	return (
+		<div className="space-y-4">
+			<ul className="grid grid-cols-1 gap-4 md:grid-cols-2">
+				{teamMembers.map(member => (
+					<li key={member.id}>
+						<Card className="flex h-full flex-col gap-4 p-4">
+							<img
+								className="h-32 w-32 rounded-full object-cover"
```
Comment (mhaidarhanif) [inline app/components/shared/team-members.tsx:17]: @fikrialwan avatar image can be a bit smaller to be "h-20 w-20" or need some sense so it's just right, not too big, not too small.

**Review by mhaidarhanif — CHANGES_REQUESTED** [review]
Adjust avatar size

File: app/components/shared/team-members.tsx (hunk 1 above)
Comment (fikrialwan) [reply app/components/shared/team-members.tsx]: I adjust avatar image to be "h-24 w-24", I think that size is not too big and not too small

**Review by mhaidarhanif — APPROVED** [review]
There's still a need to improve the schema to handle the missing fields

Next PR should replace the placeholders, using real data

**Pull request merged**